lib/pages/event_page.dart

```diff
@@ -50,67 +44,71 @@ class EventPage extends StatelessWidget {
           ),
           new SliverList(
             delegate: new SliverChildListDelegate(
-              <Widget>[
-                new ListTile(
-                  leading: const Icon(Icons.slideshow),
-                  title: new Text(event.name),
-                ),
-                new ListTile(
-                  leading: const Icon(Icons.date_range),
-                  title: new Text("$startTime$endTime"),
-                ),
-                new ListTile(
-                  leading: const Icon(Icons.location_city),
-                  title: new Text(event.place ?? ""),
-                ),
-                new Divider(height: 32.0),
-                new ListTile(
-                  leading: const Icon(Icons.add_location),
-                  title: new Text(event.address ?? ""),
-                  subtitle: new Text(event.city ?? ""),
-                ),
-                new ListTile(
-                  leading: const Icon(Icons.monetization_on),
-                  title: new Text("${event.price.toStringAsFixed(2)}\$"),
-                ),
-                new Divider(height: 32.0),
-                new Container(
-                  margin: new EdgeInsets.all(16.0),
-                  child: new Row(
-                    mainAxisAlignment: MainAxisAlignment.start,
-                    children: <Widget>[
-                      new Container(
-                        margin: new EdgeInsets.only(right: 32.0),
-                        child: new Icon(Icons.description),
-                      ),
-                      new Text(
-                        "Information",
-                        style: Theme
-                            .of(context)
-                            .textTheme
-                            .title,
-                      ),
-                    ],
-                  ),
-                ),
-                new Container(
-                  margin: new EdgeInsets.only(left: 72.0, right: 16.0),
-                  child: new Text(
-                    event.description ?? "",
-                    style: Theme
-                        .of(context)
-                        .textTheme
-                        .subhead,
-                  ),
-                ),
-                new Container(
-                  height: 64.0,
-                ),
-              ],
+              eventDetails(event, context),
             ),
           ),
         ],
       ),
     );
   }
+
+  List<Widget> eventDetails(Event event, BuildContext context) {
+    String startTime = dateTimeFormatter.format(event.start);
+    String endTime = event.end == null
```

Comment:
Nice, I like that this was moved in the relevant scope leaving the build function cleaner.